src/Unleash/Communication/UnleashApiClient.cs

```diff
@@ -211,29 +209,18 @@ public async Task<bool> RegisterClient(ClientRegistration registration, Cancella
             }
         }
 
-        public async Task<bool> SendMetrics(ThreadSafeMetricsBucket metrics, CancellationToken cancellationToken)
+        public async Task<bool> SendMetrics(MetricsBucket metrics, CancellationToken cancellationToken)
         {
-            if (metricsRequestsToSkip > metricsRequestsSkipped)
-            {
-                metricsRequestsSkipped++;
-                return false;
-            }
-
-            metricsRequestsSkipped = 0;
```

Comment:
Why are we removing this?

Reply:
I have no idea what this is, but I saw it dropped in the new method you added in [#179](https://github.com/Unleash/unleash-client-dotnet/pull/179/files#diff-e189361ec95ef479c7161915691eb64f70a0ddaa41eac30285382906250fb0b7R223-R258) so I thought it was no longer relevant. Do we still care about this? Cc: @sighphyre

Reply:
Ah, yeah, we should probably keep this for now. This is the HTTP backoff stuff we added to stop hammering Unleash when it's not ready to handle a ton of traffic I think this is dropped in #179 because that's quite old (better to say 179 didn't know that it needed it yet because it didn't exist).

This probably needs to change at some point in the near future but this might _also_ be a metrics v3 problem

Reply:
Thanks for clarifying. https://github.com/Unleash/unleash-client-dotnet/pull/224/commits/61ebe1716a59188da61aca70c763562252fac2e9 brings this back for now.